htlcswitch/link.go

```diff
@@ -3443,9 +3443,13 @@ func (l *channelLink) canSendHtlc(policy models.ForwardingPolicy,
 		return NewLinkError(&lnwire.FailTemporaryNodeFailure{})
 	}
 
-	auxBandwidth.WhenSome(func(bandwidth lnwire.MilliSatoshi) {
-		availableBandwidth = bandwidth
-	})
+	if auxBandwidth.IsHandled && auxBandwidth.Bandwidth.IsSome() {
```

Comment:
Do we really need this second arg (`auxBandwidth.Bandwidth.IsSome()`), if the some is not set we just pass the check below ?

Reply:
Yeah, this is super-defensive I guess. More for the reader of the code than the compiler. But can remove if you feel it has the opposite effect, reducing clarity.

Reply:
ahh ok yeah let's keep it then.